examples/typescript-backbone/js/app.ts

```diff
@@ -261,6 +285,28 @@ class TodoView extends Backbone.View {
 
 }
 
+// Todo Router
+// -----------
+
+class TodoRouter extends Backbone.Router {
+
+	routes = {
+		'*filter': 'setFilter'
+	};
+
+	constructor () {
+		super();
+		(<any>this)._bindRoutes();
+	}
+
+	setFilter(param: string) {
```

Comment:
oh can we use a default param here? (just noticed this)

https://github.com/tastejs/todomvc/pull/1544#discussion_r46760969